docs/.nav.yml

```diff
@@ -132,7 +132,6 @@ nav:
         - Services:
           - About Knative Services: serving/services/README.md
           - Creating a Service: serving/services/creating-services.md
-          - Service metrics: serving/services/service-metrics.md
```

Comment:
Can you *add* a redirect to `config/redirects.yml` from `serving/services/service-metrics.md` to `docs/serving/observability/metrics/serving-metrics.md` ?  That way anyone with the old link finds the new page.

Reply:
done - sorry got lost in the rebase